core/trino-main/src/test/java/io/trino/sql/query/TestSubqueries.java

```diff
@@ -1301,4 +1301,12 @@ public void testCorrelatedUnnestInLateralSubquery()
                         "ON TRUE "))
                 .matches("VALUES 1, 2, 3, 4, 5, null");
     }
+
+    @Test
+    public void testQuantifiedComparison()
```

Comment:
Is there a similar test with in-predicate? If no, could you add one?

Reply:
Added